client/src/pages/reports/Form.js

```diff
@@ -188,6 +190,40 @@ class BaseReportForm extends Component {
 				submitForm,
 				resetForm
 			}) => {
+				const attendeesFilters = {
+					recentContacts: {
+						label: 'Recent contacts',
+						searchQuery: false,
+						listName: 'personRecents',
+						listArgs: 'maxResults:6',
+					},
+					myColleagues: {
+						label: 'My colleagues',
+						searchQuery: true,
+						queryVars: {role: Person.ROLE.ADVISOR, matchPositionName: true, orgUuid: this.props.currentUser.position.organization.uuid},
```

Comment:
This assumes that the user has been assigned to a position which may not be the case

Reply:
Well, matchPositionName only means that the fulltext search should also filter on the position name. If a user doesn't have a position it is not a problem, it will still find the user. I think matchPositionName is handy in here as one can also type 
the position name in the search field of the attendees widget.